src/app/app.component.ts

```diff
@@ -58,9 +62,13 @@ export class AppComponent {
     }
   }
 
-  changeLocale(locale: string) {
-    localStorage.setItem("locale", locale);
-    location.reload();
+  @Input()
+  set language(value: string) {
```

Comment:
Would be nice to add a getter which just returns `this.localStorageLanguage`, so that external users can fetch the current language used by NGE as well.

Reply:
true !